lib/ra10ke/solve.rb

```diff
@@ -35,6 +36,9 @@ def define_task_solve_dependencies(*_args)
       end
       # Actual new logic begins here:
       cache = (ENV['XDG_CACHE_DIR'] || File.expand_path('~/.cache'))
+
+      FileUtils.mkdir_p(cache) unless Dir.exist?(cache)
```

Comment:
No need to check for existence with this function.
```suggestion
      FileUtils.mkdir_p(cache)
```

Reply:
fixed